provision.yaml

```diff
@@ -0,0 +1,6 @@
+default:
+  provisioner: vmpooler
+  images: ['win-2012r2-x86_64']
+release_checks:
+  provisioner: vmpooler
+  images: ['win-2012-x86_64', 'win-2012r2-x86_64', 'win-2016-x86_64', 'win-2019-x86_64']
```

Comment:
Although the parallel job isn't currently working, why do we not supply all supported OS for release_checks?